.github/workflows/build.yml

```diff
@@ -42,6 +43,7 @@ jobs:
           DM_CONNECTION_STRING: "https://ingest-sdkse2etest.eastus.kusto.windows.net"
           AZURE_CLIENT_ID: ${{ secrets.APP_ID }}
           AZURE_TENANT_ID: ${{ secrets.TENANT_ID }}
+          NODE_OPTIONS: "--experimental-vm-modules"
```

Comment:
Needed for jest to support esm, we may want to move to vitest in the future (if we adopt the vite ecosystem)